templates/careers/index.html

```diff
@@ -37,10 +37,20 @@ <h2 class="p-text--x-small-capitalised u-align-text--x-small-to-default">Departm
   </div>
   <div class="row">
     {% for dept in departments_overview %}
-      <div class="col-3">
-        <a class="p-heading--5 u-no-margin--bottom" href="/careers/{{dept.slug}}">{{ dept.name }}</a>
-        <p>{{ dept.count }} roles</p>
+    <div class="col-3 col-medium-2 col-small-2">
+      <div class="p-media-object">
+        <img src="https://assets.ubuntu.com/v1/{{dept.icon}}" class="p-media-object__image" style="width: 32px; height: 32px; position: relative; top: 7px;" alt="">
+        <div class="p-media-object__details">
+          <h3 class="p-media-object__title p-heading--5">
```

Comment:
`p-heading--5` matches the design. The `p` looks huge:
![image](https://user-images.githubusercontent.com/1413534/220582969-81822905-f7b2-455e-a4b1-a3f446bf61da.png)

Reply:
hmm I think the class overrides the font properties. Ok, fint to have it as an h5, but we need the roles closer, as they would be if both pieces of text were in the same element, separated by a `<br>`. The roles shouldn't be bold. One quick and dirty way to achieve this would be to drop the `p-media-object__title` maybe? Happy to jump on a call and explain if the above is confusing.